login.py

```
@@ -3,6 +3,7 @@
 
 def login(username, password):
     conn = sqlite3.connect("users.db")
+    print ("Connection string", conn)
     cursor = conn.cursor()
     query = f"SELECT * FROM users WHERE username='{username}' AND password='{password}'"
```

Ruff (0.14.8)

Possible SQL injection vector through string-based query construction

(S608)